portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,9 +32,8 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     ArrayList<String> messages = new ArrayList<String>();
-    messages.add("Hello");
-    messages.add("Goodbye");
-    messages.add("See you later");
+    String comment = getParameter(request, "textInput", "nothing");
+    messages.add(comment);
```

Comment:
Remove these lines. This is the operation done by doPost(). All doGet() needs to do is the rest of this method that you already have.